mne_bids/write.py

```diff
@@ -300,20 +301,25 @@ def _participants_tsv(raw, subject_id, fname, overwrite=False,
             if isinstance(meas_date, datetime):
                 meas_datetime = meas_date
             else:
-                meas_datetime = datetime.fromtimestamp(meas_date)
+                meas_datetime = datetime.fromtimestamp(meas_date, tz=timezone.utc)
```

Comment:
why was adding a timezone necessary?

Reply:
yes, sorry for being unclear:
this is actually unrelated to the handedness information..

i saw that the datetime object for the participants birthdate has a timezone, while it was not specified for the date of recording. 
in my testing without the edit i got the following error:
```
File "/Users/dominik.welke/Work/git_contributions/mne-bids/mne_bids/write.py", line 1241, in write_raw_bids
    verbose)
  File "/Users/dominik.welke/Work/git_contributions/mne-bids/mne_bids/write.py", line 305, in _participants_tsv
    subject_age = _age_on_date(bday, meas_datetime)
  File "/Users/dominik.welke/Work/git_contributions/mne-bids/mne_bids/utils.py", line 352, in _age_on_date
    if exp_date < bday:
TypeError: can't compare offset-naive and offset-aware datetimes
```